src/platform/posix.c

```diff
@@ -818,26 +817,13 @@ time_t platform_file_get_modified_time(const utf8* path){
 uint8 platform_get_locale_currency(){
 	char *langstring = setlocale(LC_MONETARY, "");
 
-	if(langstring != NULL){
-		struct lconv *lc = localeconv();
-
-		//Only works if g_currency_specs contains the actual (local) symbol
-		for(int i = 0; i < CURRENCY_END; ++i){
-			if(!strcmp(lc->currency_symbol, CurrencyDescriptors[i].symbol_unicode)){
-				return i;
-			}
-		}
-		//TODO: can be removed when CurrencyDescriptors contains the actual symbols for won and rubel
-		//Won should remain a special case, beacause some (or all?) systems use the full width won sign (e.g. Gentoo)
-		if(!strncmp(lc->int_curr_symbol, "KRW", 3)){
-			return CURRENCY_WON;
-		}
-		else if(!strncmp(lc->int_curr_symbol, "RUB", 3)){
-			return CURRENCY_ROUBLE;
-		}
+	if (langstring == NULL) {
```

Comment:
I think it gets triggered when `LANG` is set to invalid value. Try `$ LANG=0123456 ./openrct2`. `lc->int_curr_symbol` is not referenced at this point, but it's safe the way you handle it here.